src/graphics.rs

```diff
@@ -116,7 +116,8 @@ where
 }
 
 impl BitsPerPixel {
-    pub const fn from_rgbcolor<C: RgbColor>() -> Self {
+    /// Returns the bits per pixel for a embedded-graphics [`RgbColor`].
+    pub const fn from_rgb_color<C: RgbColor>() -> Self {
```

Comment:
Not related to the docs, but at least IMO more readable and the default conversion from camel case to snake case.